src/index.html

```diff
@@ -5,9 +5,42 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+
+    <div class="card" data-qa="card">
+      <img
```

Comment:
<img width="1331" alt="Screenshot 2023-06-01 at 17 21 15" src="https://github.com/mate-academy/layout_product-cards/assets/53228259/fa963277-abbd-4e3c-bc0b-90b3fd8feb0e">

Reply:
Not fixed

Reply:
still  not fixed